helm/cfgov/templates/tests/smoke-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "smoke-test-job"
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+    metadata:
+      name: "smoke-test-job"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
```

Comment:
Just use the `app.labels` template (see other tests), no need to add them each in individually.